pkg/cluster/space/rook.go

```diff
@@ -0,0 +1,154 @@
+package clusterspace
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"code.cloudfoundry.org/bytefmt"
+	rookcli "github.com/rook/rook/pkg/client/clientset/versioned"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"github.com/replicatedhq/kurl/pkg/k8sutil"
+)
+
+const (
+	namespace = "rook-ceph"
+)
+
+// RookChecker checks if we have enough disk space to migrate volumes to rook.
+type RookChecker struct {
```

Comment:
```suggestion
type RockDiskSpaceValidator struct {
```

wdyt about something like ^

Reply:
I don't like it. `clusterspace.RookDiskSpaceValidator` does look too repetitive.

Reply:
Will this check check anything else?
What about only DiskSpaceValidator or DiskSpaceChecker?

My idea here were we have a more meaningful name about what it actually does.

Reply:
fixed.